src/main/resources/locales/cy/translation/application-details.json

```
@@ -8,6 +8,7 @@
   "Claimant": "Hawlydd",
   "Respondent": "Atebydd",
   "Respondent Representative": "Cynrychiolydd yr Atebydd",
+  "Claimant Representative": "<placeholder>",
```

Got this from et-syr
```suggestion'
  "Claimant Representative": "Cynrychiolydd yr Hawlydd",
```